docs/builds/guides/frameworks/vuejs.md

```diff
@@ -38,21 +38,29 @@ Optionally, you can [use the component locally](#using-component-locally).
 
 ### Direct script include
 
-This is the quickest way to start using CKEditor in your project. Assuming [Vue is installed](https://vuejs.org/v2/guide/installation.html), include the `<script>` tags for the WYSIWYG editor component and the build:
+This is the quickest way to start using CKEditor in your project. Assuming Vue is installed (check out the official guides for [Vue 2.x](https://vuejs.org/v2/guide/installation.html) and for [Vue 3.x](https://v3.vuejs.org/guide/installation.html) to learn more), include the `<script>` tags for the WYSIWYG editor component and the build:
 
 ```html
 <script src="../node_modules/@ckeditor/ckeditor5-build-classic/build/ckeditor.js"></script>
 <script src="../node_modules/@ckeditor/ckeditor5-vue/dist/ckeditor.js"></script>
 ```
 
-Enable the component in your application by using the [`Vue.use()`](https://vuejs.org/v2/api/#Vue-use) method:
+If you are using Vue 2.x, enable the component globally in your application by using the [`Vue.use()`](https://vuejs.org/v2/api/#Vue-use) method:
 
 ```js
-Vue.use( CKEditor );
+// Vue 2.x only.
+Vue.use( CKEditor( Vue ) );
+```
+
+If you have Vue 3.x, use the [application instance](https://v3.vuejs.org/guide/instance.html#creating-an-application-instance) to install the component globally:
+
+```js
+// Vue 3.x only.
+Vue.createApp( { /* options */ } ).use( CKEditor( Vue ) );
```

Comment:
In https://github.com/ckeditor/ckeditor5/pull/8289/files#diff-b2ab47c3d98e0c613d60703d6ef661ade227658ea9062e1e0378cc853ad17904R160 you state that `h()` is mandatory. Which way is it?